pkg/csi/service/wcp/controller.go

```diff
@@ -511,8 +496,9 @@ func (c *controller) createBlockVolume(ctx context.Context, req *csi.CreateVolum
 		VsanDirectDatastoreURL: selectedDatastoreURL,
 	}
 	candidateDatastores := append(sharedDatastores, vsanDirectDatastores...)
+	useSupervisorId := commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.TKGsHA)
```

Comment:
Do you want to set `useSupervisorId` in `Init` and use that in every CreateVolume call?

Reply:
utilized the check we used for making changes in the CreateVolume Call

```
	isTKGSHAEnabled := commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.TKGsHA)
	if isTKGSHAEnabled {
		// TKGS-HA feature is enabled
		// Identify the topology keys in Accessibility requirements.
```